package.json

```diff
@@ -22,7 +22,8 @@
   ],
   "scripts": {
     "test": "echo \"No test specified\"",
-    "start:dev": "npx nodemon"
+    "start:dev": "npx nodemon",
+    "build": "tsc --outDir ./dist"
```

Comment:
I suggest removing the outDir and leave it for tsc to figure out where the outDir is, from the tsconfig.json.